frame/babe/src/equivocation.rs

```diff
@@ -181,6 +181,11 @@ impl<T: Config> frame_support::unsigned::ValidateUnsigned for Module<T> {
 				}
 			}
 
+			if let Call::report_equivocation_unsigned(equivocation_proof, key_owner_proof) = call {
```

Comment:
why are you not using the data (i.e `equivocation_proof `) of the previous `if let Call::` ?

Reply:
🤦 Because it was already bedtime when I wrote this. Updated.